code/modules/projectiles/ammo_datums.dm

```diff
@@ -3578,7 +3578,7 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	drop_resin(T.density ? P.loc : T)
 
 /datum/ammo/xeno/sticky/proc/drop_resin(turf/T)
-	if(T.density)
+	if(T.density || istype(T, /turf/open/space)) // No structures in space
```

Comment:
So, there's also `is_weedable()`, but that would disallow resin spit on grass and coast as well. The full `is_valid_for_resin_structure()` isn't really applicable here either.

Reply:
you can just make a is_resinable() proc or something. Honestly though, space tiles were not intended for actual use in game, thats the real issue.